src/software/constants.h

```diff
@@ -49,6 +49,7 @@ const std::string ROBOT_KICK_CONSTANT_REDIS_KEY     = "/kick_constant";
 const std::string ROBOT_CHIP_PULSE_WIDTH_REDIS_KEY  = "/chip_pulse_width";
 const std::string ROBOT_CURRENT_DRAW_REDIS_KEY      = "/current_draw";
 const std::string ROBOT_BATTERY_VOLTAGE_REDIS_KEY   = "/battery_voltage";
+const std::string CAPACITOR_VOLTAGE_REDIS_KEY       = "/cap_voltage";
```

Comment:
```suggestion
const std::string ROBOT_CAPACITOR_VOLTAGE_REDIS_KEY = "/cap_voltage";
```
Can we change this to match the other variable names in the file